ST_main.c

```diff
@@ -440,7 +497,6 @@ static void init_args(int argc, char **argv) {
   /* Defaults */
   parm_SetFirstName( DFLT_FIRSTFILE);
   QuietMode = EchoInits = UseSeedDispersal = FALSE;
-  SXW.debugfile = NULL;
```

Comment:
Since I changed SXW to a pointer it is not allocated when this function runs. This line has been moved to SXW_Init right after the memory is allocated.